images/win/scripts/Tests/Haskell.Tests.ps1

```diff
@@ -0,0 +1,33 @@
+Describe "Haskell" {
+    It "ghc" {
+        "ghc --version" | Should -ReturnZeroExitCode
+    }
+
+    It "cabal" {
+        "cabal --version" | Should -ReturnZeroExitCode
+    }
+
+    $chocoPackagesPath = Join-Path $env:ChocolateyInstall "lib"
+    [array]$ghcVersionList = Get-ChildItem -Path $chocoPackagesPath -Filter "ghc.*" | ForEach-Object { $_.Name.TrimStart("ghc.") }
+    $ghcCount = $ghcVersionList.Count
+
+    It "Accurate 3 versions of GHC are installed" -TestCases @{ghcCount = $ghcCount} {
+        $ghcCount | Should -BeExactly 3
+    }
+
+    $ghcTestCases = $ghcVersionList | ForEach-Object {
+        @{
+            ghcVersion = $_
+        }
+    }
+
+    It "Validation each of GHC version" -TestCases $ghcTestCases {

```

Comment:
This line will produce 3 tests with the same name.
Does it make sense to write `It "<ghcVersion> is installed"`